### DISCUSSION TITLE
Defective Turrets Voice

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#3866] [Feature Suggestions]
Could you guys add voices from the defective turret like on the 12 Angry Tests maps?

**Comment by vrad-exe** [comment]
Why would that make any sense?

**Comment by FrozenDragon0** [comment]
No, that is something you should be using hammer for. Also the defective turret doesn't even have that many voicelines for something story related to begin with.

**Comment by CN1o2** [comment]
I really don't know how to explain it, I just THOUGHT it would be a good idea.
